src/Control/Email/Email.php

```diff
@@ -316,430 +221,142 @@ private function getDefaultFrom(): string
         return $defaultFrom;
     }
 
-    /**
-     * @return string[]
-     */
-    public function getFrom()
-    {
-        return $this->getSwiftMessage()->getFrom();
-    }
-
-    /**
-     * @param string|array $address
-     * @return string|array
-     */
-    private function sanitiseAddress($address)
+    public function setBody(AbstractPart|string $body = null): static
     {
-        if (is_array($address)) {
-            return array_map('trim', $address ?? []);
+        if ($body instanceof AbstractPart) {
+            // pass to Symfony\Component\Mime\Message::setBody()
+            return parent::setBody($body);
         }
-        return trim($address ?? '');
+        // backwards compat with 4.x
+        return $this->html($body);
     }
 
-    /**
-     * @param string|array $address
-     * @param string|null $name
-     * @return $this
-     */
-    public function setFrom($address, $name = null)
+    public function setFrom(string $address, string $name = ''): static
     {
-        $address = $this->sanitiseAddress($address);
-        $this->getSwiftMessage()->setFrom($address, $name);
-
-        return $this;
+        return $this->from(new Address($address, $name));
     }
 
-    /**
-     * @param string|array $address
-     * @param string|null $name
-     * @return $this
-     */
-    public function addFrom($address, $name = null)
+    public function setSender(string $address, string $name = ''): static
     {
-        $address = $this->sanitiseAddress($address);
-        $this->getSwiftMessage()->addFrom($address, $name);
-
-        return $this;
+        return $this->sender(new Address($address, $name));
     }
 
-    /**
-     * @return string
-     */
-    public function getSender()
+    public function setTo(string $address, $name = ''): static
     {
-        return $this->getSwiftMessage()->getSender();
+        return $this->to(new Address($address, $name));
     }
 
-    /**
-     * @param string $address
-     * @param string|null $name
-     * @return $this
-     */
-    public function setSender($address, $name = null)
+    public function setCC(string $address, string $name = ''): static
     {
-        $address = $this->sanitiseAddress($address);
-        $this->getSwiftMessage()->setSender($address, $name);
-
-        return $this;
+        return $this->cc(new Address($address, $name));
     }
 
-    /**
-     * @return string
-     */
-    public function getReturnPath()
+    public function setBCC(string $address, string $name = ''): static
     {
-        return $this->getSwiftMessage()->getReturnPath();
+        return $this->bcc(new Address($address, $name));
     }
 
-    /**
-     * The bounce handler address
-     *
-     * @param string $address Email address where bounce notifications should be sent
```

Comment:
I'm not a fan of maintaining docs that describe something that's already obvious based on the method name.

Also, the set* methods are not just aliases retain for backward compatibility. The actual method `replyTo` also doesn't have PHP docs describing it - https://github.com/symfony/mime/blob/6.1/Email.php#L144

Reply:
> I'm not a fan of maintaining docs that describe something that's already obvious based on the method name

Fair enough - I just wanted to make sure it wasn't removed accidentally since there was no clear reason to remove it and you hadn't made any comment indicating the reason. I'm okay with this being removed given "replyTo" is fairly obvious.

> The actual method replyTo also doesn't have PHP docs describing it

That's irrelevant, we're not the ones implementing that method.